org.gnome.Sudoku.json

```diff
@@ -25,8 +25,8 @@
             "buildsystem": "meson",
             "sources": [{
                 "type": "archive",
-                "url": "https://download.gnome.org/sources/gnome-sudoku/42/gnome-sudoku-42.0.tar.xz",
-                "sha256": "1d2eb4ddb8026b443645cf3585b8df1244e3828ee1c07518052b2599e1c5c28f"
+                "url": "https://download.gnome.org/sources/gnome-sudoku/44/gnome-sudoku-44.0.tar.xz",
+                "sha256": "6518d9233a46d7e13805ac787667ba4701948d9ed41a47b88797fcdba43b8fba"
```

Comment:
Let's add external data checker support here so it doesn't keep getting out of date again. I just copied this from org.gnome.Chess which seems to work:

```
                "x-checker-data": {
                    "type": "json",
                    "url": "https://download.gnome.org/sources/gnome-sudoku/cache.json",
                    "version-query": ".[2][\"gnome-sudoku\"] | map(select(. | match(\"([0-9].[0-9]*[02468].[0-9]+)|([0-9]+.[0-9]+)\"; \"g\"))) | last",
                    "url-query": "\"https://download.gnome.org/sources/gnome-sudoku/\" + .[1][\"gnome-sudoku\"][$version][\"tar.xz\"]"
                }
```

Reply:
We already have a specific gnome checker..  Please use that one instead

Reply:
OK, I found an example:

```
                    "x-checker-data": {
                        "type": "gnome",
                        "name": "gnome-sudoku"
                    }
```

Reply:
Oh and @bilelmoussaoui could you give me access to this repo so I can merge the change please? Thanks.